### PR TITLE
Unsafe chapter in spanish

### PR DESCRIPTION
**Pull request by ericho**

### CODE REVIEW

**Comment by Hoverbear** [comment]
@emig Can you review this? :)

File: presentation/chapters/es-ES/unsafe.chapter [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+El sistema de tipo de Rust provee muchas garantías, pero en ocasiones esto conlleva que soluciones específicas sean difíciles o imposibles.
+
+Por esta razón, Rust tiene el concepto de "código inseguro".
+
+---
+
+El código inseguro tiene permitido:
+-   acceder memoria libremente
+-   utilizar apuntadores crudos (raw pointers)
```
Comment (emig) [inline presentation/chapters/es-ES/unsafe.chapter]: I think we use more the term `punteros` than `apuntadores` see https://es.wikipedia.org/wiki/Puntero_(inform%C3%A1tica)

File: presentation/chapters/es-ES/unsafe.chapter [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+-   llamar a funciones externas
+-   declarar valores `Send` y `Sync`
+-   escribir a variables globales no sincronizadas
+
+---
+
+No es seguro:
+
+-   conversión a apuntadores crudos
```
Comment (emig) [inline presentation/chapters/es-ES/unsafe.chapter]: `crudo` is too related to food, I would take this option https://es.wikipedia.org/wiki/Puntero_(inform%C3%A1tica)#Haciendo_m.C3.A1s_seguros_a_los_punteros, (although the page indicates translation is not accurate :)). They use `punteros sin procesar`:

>>> En este contexto, los punteros que se dirigen directamente a la memoria (como se utilizan en este artículo) se denominan punteros sin procesar, en contraste con los punteros inteligentes u otras variantes

File: presentation/chapters/es-ES/unsafe.chapter [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+-   pérdidas de memoria (memory leaks)
```
Comment (emig) [inline presentation/chapters/es-ES/unsafe.chapter]: https://es.wikipedia.org/wiki/Fuga_de_memoria  sounds better

File: presentation/chapters/es-ES/unsafe.chapter [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+## El pequeño secreto de Rust
+
+Cuando se implementan estructuras de datos, los bloques unsafe no son inusuales.
+
+Rust es el peor lenguaje para implementar listas ligadas. Hay un [texto completo](TODO:%20Link) sobre este tema.
+
+---
+
+El codigo inseguro debe *siempre* ser marcado como `unsafe`.
```
Comment (emig) [inline presentation/chapters/es-ES/unsafe.chapter]: código

File: presentation/chapters/es-ES/unsafe.chapter [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+<pre><code data-source="chapters/shared/code/unsafe/1.rs" data-trim="hljs rust" class="lang-rust"></code></pre>
+
+---
+
+## Trampas de `unsafe`
+
+-   No todos los ejemplos son tan simples. `unsafe` *debe* garantizar las invariantes que Rust espera.
+-   Esto *especialmente* aplica a la propiedad y préstamo mutable.
```
Comment (emig) [inline presentation/chapters/es-ES/unsafe.chapter]: Esto es *especialmente* aplicable a la propiedad y préstamo mutable.

File: presentation/chapters/es-ES/unsafe.chapter [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+---
+
+Rust permite que te dispares en el pie, solo requiere que remuevas el seguro de tu arma antes de sacarla de la funda.
+
+---
+
+## Ejemplo práctico
+
+Como Rust prohibe el "aliasing", es imposible en Rust seguro separar un slice en 2 partes sin superposició.
```
Comment (emig) [inline presentation/chapters/es-ES/unsafe.chapter]: superposición

**Comment by emig** [comment]
@Hoverbear done, @ericho thanks!

File: presentation/chapters/es-ES/unsafe.chapter (hunk 1 above)
Comment (ericho) [inline presentation/chapters/es-ES/unsafe.chapter]: Thanks :smile: is fixed now.

File: presentation/chapters/es-ES/unsafe.chapter (hunk 2 above)
Comment (ericho) [inline presentation/chapters/es-ES/unsafe.chapter]: This was hard to translate :) It's a good idea to use `sin procesar` as translation of `raw`.

File: presentation/chapters/es-ES/unsafe.chapter (hunk 3 above)
Comment (ericho) [inline presentation/chapters/es-ES/unsafe.chapter]: Fixed :)

**Comment by ericho** [comment]
Thanks @emig for the review. Sorry about the delay, I was stuck at work the last weeks.

**Pull request merged**